ssl/quic/qlog_event_helpers.c

```diff
@@ -213,8 +213,11 @@ static int log_frame_actual(QLOG *qlog_instance, PACKET *pkt,
     size_t i;
     PACKET orig_pkt = *pkt;
 
-    if (!ossl_quic_wire_peek_frame_header(pkt, &frame_type, NULL))
+    if (!ossl_quic_wire_peek_frame_header(pkt, &frame_type, NULL)) {
+        if (*need_skip == 0)
+            *need_skip = 1;
```

Comment:
Actually the correct thing here is:
```suggestion
            *need_skip = PACKET_remaining(pkt);
```
So we basically skip to the end of the packet if we cannot parse it as quic frame(s).

Reply:
When you want to really fix the qlog, then a hex-dump of the packet would be necessary.
But I have spent enough time on this useless feature already.

Reply:
Could you please, please push this change as I've requested so I can approve it and it can be merged? I would push a fixup commit to your branch but you disallowed that.

Reply:
> When you want to really fix the qlog, then a hex-dump of the packet would be necessary.

Not really. If a packet is just a garbage, I do not think it needs to be properly qlogged.

Reply:
Hmm, maybe I have a better idea, just set *need_skip = SIZE_MAX; then it is really the last one.

Reply:
Hmm, that does not work because there is another bug on line 547 - the condition is inverted there.

Reply:
I dont see any problem there, the packet is consumed, and the following packets as well.
That is my intention.